hooks/boot/01-helpers.rb

```diff
@@ -10,7 +10,7 @@ def module_enabled?(kafo, name)
       mod.enabled?
     end
 
-    def log_and_say(level, message)
+    def log_and_say(level, message, log_only = false)
```

Comment:
@ekohl so, the problem there is the function is named `say` which means I can't name the variable that as well.